dotCMS/src/main/java/com/dotmarketing/cms/content/submit/action/SubmitContentAction.java

```diff
@@ -57,7 +58,8 @@
  * @author Oswaldo
  *
  */
-public class SubmitContentAction extends DispatchAction{
+@Deprecated
+public class SubmitContentAction extends SecureAction{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)